### ISSUE TITLE
Problems with inserting negative integers

### ISSUE THREAD

**Issue by pnalysz** [JuliaDatabases/ODBC.jl#161]
Hi,

I am not able to insert negative integers to the table with column of 'int' data type. I am using Microsoft SQL Server but I had the same problem with PostgreSQL.

Here is my script:
using ODBC
dsn = ODBC.DSN("msqls_local")
sql = "INSERT INTO [test].[dbo].[test] ([integer]) VALUES (?)"
statement = ODBC.prepare(dsn, sql)
data = [
  (1,),
  (-2,)
]
for record in data
  dump(record)
  ODBC.execute!(statement, record)
end

And here is the console output:
Tuple{Int64}
  1: Int64 1
Tuple{Int64}
  1: Int64 -2
[ODBC] 22003: [Microsoft][ODBC Driver 13 for SQL Server]Numeric value out of range

Value '1' goes in without any problem but for any negative integer I get the error.

Thanks in advance,
Piotr

**Comment by quinnj** [comment]
Hmmmm, this is weird, but I'm also not sure this is a package problem, or some kind of weird data + database settings issue (many google search results seem to point to the latter). Can you try on current ODBC master as there have been a big round of updates just merged in?

**Comment by takkaaa** [comment]
Hi,
After `Pkg.checkout("ODBC")` I get:
```
INFO: Checking out ODBC master...
INFO: Pulling ODBC latest master...
ERROR: Unsatisfiable requirements detected for package WeakRefStrings:
├─version range [0.3.0,∞) required by package ODBC, whose only allowed version is 0.5.2+:
│ ├─version 0.5.2+ set by fixed requirement (package is checked out, dirty or pinned)
│ └─version range [0.0.0-,∞) set by an explicit requirement
└─version range [0.0.0,0.3.0) required by package CSV, whose allowed version range is [0.0.0-,∞):
  └─version range [0.0.0-,∞) set by an explicit requirement
The intersection of the requirements is empty.
```
Sorry top bother you if it is something obvious but I am not yet very advanced julia user.

Thanks,
Piotr

**Comment by takkaaa** [comment]
Ok, I managed upgrade ODBC to 0.5.2+ and now I get error that @isdefined is not defined. Does it mean that I have to test it in Julia v0.7?

**Comment by quinnj** [comment]
This should have been fixed in #263
